include/vctr/Containers/VctrBase.h

```diff
@@ -338,25 +338,29 @@ class VctrBase : private Config,
     // Finding elements and manipulating them
     //==============================================================================
     /** Returns an iterator to the first element that equals valueToLookFor or end() if none was found */
-    constexpr auto find (const ElementType& valueToLookFor)
+    template <std::equality_comparable_with<ElementType> T>
+    constexpr auto find (const T& valueToLookFor)
     {
         return std::find (begin(), end(), valueToLookFor);
     }
 
     /** Returns a const iterator to the first element that equals valueToLookFor or end() if none was found */
-    constexpr auto find (const ElementType& valueToLookFor) const
+    template <std::equality_comparable_with<ElementType> T>
+    constexpr auto find (const T& valueToLookFor) const
     {
         return std::find (begin(), end(), valueToLookFor);
     }
 
     /** Returns a reverse iterator to the last element in this vector that equals p or rend() if none was found */
```

Comment:
I think this comment should read "that equals valueToLookFor" (instead of "equals p") – also the next comment below.

Reply:
Ooops, copy & paste error from the old implementation...